src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -39,6 +42,7 @@ public ModelManager(ReadOnlyAddressBook addressBook, SalesBook salesBook, ReadOn
         this.salesBook = new SalesBook(salesBook);
         this.userPrefs = new UserPrefs(userPrefs);
         filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        filteredIngredients = new FilteredList<>(this.addressBook.getIngredientList());
```

Comment:
As mentioned earlier, functions and features relating to ingredients tracking should be abstracted out to its own classes (instead of being in AddressBook).